docs/user-guide/mc.ipynb

```diff
@@ -1,885 +0,0 @@
-    "So we work around this by storing a 2D array with artificial dimension labels (`dims`)."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "423d3bdc-ff0a-41c6-bd20-93bff4210d18",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "variance_accumulator = mc.VarianceAccum()\n",
-    "covariance_accumulator = mc.CovarianceAccum(dims=(\"x0\", \"x1\"))"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "5fb0ff44-fd4b-43b8-a68e-bb9cd354bc1b",
-   "metadata": {},
-   "source": [
-    "In order to use this, we need to return an additional result from our speed function.\n",
-    "Since we want both the mean+variance and covariance matrix of the same result, the speed, we simply return it twice."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "4fd90b82-9ead-4d1f-9041-022bf302f95e",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "def compute_speed_cov(\n",
-    "    pos: sc.DataArray, time: sc.DataArray\n",
-    ") -> Dict[str, sc.DataArray]:\n",
-    "    speed = pos / time\n",
-    "    return {\"speed\": speed, \"speed_cov\": speed}"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "dce32218-55a5-49c5-9a36-08c311ad3b39",
-   "metadata": {},
-   "source": [
-    "Now, run the MC calculation:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "676b2a82-0dd7-4f90-9517-8ae54d4974a6",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "results = mc.run(\n",
-    "    compute_speed_cov,\n",
-    "    samplers={\"pos\": pos_sampler, \"time\": time_sampler},\n",
-    "    accumulators={\"speed\": variance_accumulator,\n",
-    "                  \"speed_cov\": covariance_accumulator},\n",
-    "    n_samples=100_000,\n",
-    "    progress=False,\n",
-    ")\n",
-    "speed_mc = results[\"speed\"]\n",
-    "speed_cov = results[\"speed_cov\"]"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "e2b0fdeb-cc5f-402a-851a-cd1ac9d66f61",
-   "metadata": {},
-   "source": [
-    "`speed_mc` is the same as before.\n",
-    "And `speed_cov` holds the variance-covariance matrix:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "5f214c3a-c08c-4cf7-a34e-f9dd1ae2e007",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "pp.plot(speed_cov)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "24fe86af-8dbd-455d-b9d4-4539469009be",
-   "metadata": {},
-   "source": [
-    "The variance-covariance matrix is difficult to interpret.\n",
-    "So compute the linear correlation matrix:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "c8fd5967-8064-4b4d-978b-685fe9b82c0d",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "def pearson_correlation(cov: sc.DataArray) -> sc.DataArray:\n",
```

Comment:
Can the `pearson_correlation` function be a generic function that is part of the module, so that we don't need to define it in the notebook but just import it from the package?